### DISCUSSION TITLE
Handle timezones in date fields

### DISCUSSION THREAD

**Discussion by pehbehbeh** [naymspace/backpex#334] [Ideas]
Time zones should be taken into account for the date fields.

**Possible solutions:**

- Define time zone on the field. (Con: Time can only ever be displayed in a defined time zone.)
- Always convert time to local time. (Con: You don't always want to see times only in local time.)

**Comment by AlexWiles** [comment]
I am trying out Backpex on a project and need to show the timezone based on the current user. 

I solved this by tweaking the existing DateTime field to take a timezone_fn that receives assigns and returns a timezone.
